src/docs/quick-start/test-wasm-wrappers/infra-pipeline.md

```diff
@@ -0,0 +1,124 @@
+modules as directed by the user.
+
+## Declaration
+
+Unlike some manifests, the Infra manifest does not need to be declared in your Polywrap manifest.
+
+## Content
+
+The Infra manifest consists of environmental variable declarations and one or more infrastructure modules. 
+Each module points to a local, remote, or default docker-compose file.
+
+<Tabs
+defaultValue="schema"
+values={[
+{label: 'Schema', value: 'schema'},
+{label: 'Example', value: 'example'},
+]}>
+<TabItem value="schema">
+
+```yaml
+format: # The manifest format version
+env: # Declare environmental variables here
+modules:
+  myRemote: # A remote package with a docker-compose file
+    package: # Package name
+    version: # Package version
+    registry: # Package registry name
+    dockerComposePath: # (Optional) Path to docker-compose file in the package directory
+  myLocal: # A local package with a docker-compose file
+    path: # Path to the package
+  eth-ens-ipfs: default # A module available by default
+```
+
+</TabItem>
+<TabItem value="example">
+
+```yaml
+format: 0.1.0
+env:
+  IPFS_PORT: $ENV_IPFS_PORT
+  DEV_SERVER_PORT: 4040
+  DEV_SERVER_ETH_TESTNET_PORT: 8545
+modules:
+  ipfs:
+    package: "@namestys/ipfs-infra"
+    version: "0.0.1"
+    registry: npm
+  dev-server:
+    path: ../local-packages/dev-server
+```
+</TabItem>
+</Tabs>
+
+## Infrastructure Modules
+
+An Infra manifest can declare any number of infrastructure modules. 
+Polywrap currently supports three types of infrastructure modules: 
+- A local module exists on your local filesystem.
+- A remote module is a package hosted by a package registry.
+- The default module is included with the CLI.
+
+### Local
+
+A local infrastructure module is a path from the Infra manifest to a local folder with a docker-compose file.
+
+```yaml title="Example: local module configuration"
+format: 0.1.0
+modules:
+  myLocal:
+    path: ../local-packages/myLocal
+```
+
+### Remote
+
+A remote infrastructure module is a package hosted at a package registry. 
+The package must contain a docker-compose file. 
+The path to the docker-compose file must be declared in the Infra manifest if the file is not located in the package root.
+
+Remote packages can be shared. 
+Users can add remote packages to their manifest to replicate the infrastructure modules defined by other users or projects.
+
+```yaml title="Example: remote module configuration"
+format: 0.1.0
+modules:
+  myIpfsNode:
+    package: "@namestys/ipfs-node"
+    version: "1.0.2"
+    registry: npm
+    dockerComposePath: ./config/docker-compose.yaml
+```
+
+### Default
+
+A default infrastructure module is included with the [`polywrap`](../../reference/cli/polywrap-cli) CLI. 
+It is declared in the Infra manifest as a module named *eth-ens-ipfs* and the value *default*. 
+If an Infra manifest is not found, the Polywrap CLI [`infra`](../../reference/cli/commands/deploy) command will use
```

Comment:
IIRC we need to pass the `--modules` flag if no manifest is provided :thinking: @namesty can you confirm this mi bro pl0x

Reply:
You're right. I fixed it!